syntax/scan.go

```diff
@@ -770,6 +770,17 @@ func (sc *scanner) scanNumber(val *tokenValue, c rune) Token {
 				sc.readRune()
 				c = sc.peekRune()
 			}
+		} else if c == 'b' || c == 'B' {
```

Comment:
Could you add some tests to syntax/scan_test.go (for the scanner) and testdata/int.sky for the int(x) function?